### PR TITLE
 TIP-860: Move normalizer from enrich bundle to the right BC 

### PR DESCRIPTION
**Pull request by arnolanglade**
**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | yes
| Review and 2 GTM                  | todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: CHANGELOG-3.0.md [hunk 1]
```diff
@@ -873,3 +873,23 @@
 - Rename `Pim\Bundle\FilterBundle\Filter\CompletenessFilter` to `Oro\Bundle\PimFilterBundle\Filter\ProductCompletenessFilter`
 - Move `Pim\Bundle\PimDataGridBundle` to `Oro\Bundle\PimDataGridBundle`
 - Move `Pim\Bundle\PimFilterBundle` to `Oro\Bundle\PimFilterBundle`
+- Move `Pim\Bundle\EnrichBundle\Normalizer\CategoryNormalizer` to `Akeneo\Pim\Enrichment\Component\Product\Normalizer\InternalApi\CategoryNormalizer`
```
Comment (jjanvier) [inline CHANGELOG-3.0.md]: Akeneo\Pim\Enrichment\Component\Category?

File: CHANGELOG-3.0.md [hunk 2]
```diff
@@ -873,3 +873,23 @@
...
+- Move `Pim\Bundle\EnrichBundle\Normalizer\ImageNormalizer` to `Akeneo\Pim\Enrichment\Component\Product\Normalizer\InternalApi\ImageNormalizer`
+- Move `Pim\Bundle\EnrichBundle\Normalizer\IncompleteValuesNormalizer` to `Akeneo\Pim\Enrichment\Component\Product\Normalizer\InternalApi\IncompleteValuesNormalizer`
+- Move `Pim\Bundle\EnrichBundle\Normalizer\ProductModelNormalizer` to `Akeneo\Pim\Enrichment\Component\Product\Normalizer\InternalApi\ProductModelNormalizer`
+- Move `Pim\Bundle\EnrichBundle\Normalizer\ProductNormalizer` to `Akeneo\Pim\Enrichment\Component\Product\Normalizer\InternalApi\ProductNormalizer`
+- Move `Pim\Bundle\EnrichBundle\Normalizer\ProductViolationNormalizer` to `Akeneo\Pim\Enrichment\Component\Product\Normalizer\InternalApi\ProductViolationNormalizer`
+- Move `Pim\Bundle\EnrichBundle\Normalizer\VariantNavigationNormalizer` to `Akeneo\Pim\Enrichment\Component\Product\Normalizer\InternalApi\VariantNavigationNormalizer`
+- Move `Pim\Bundle\EnrichBundle\Normalizer\VersionNormalizer` to `Akeneo\Pim\Enrichment\Component\Product\Normalizer\InternalApi\VersionNormalizer`
```
Comment (jjanvier) [inline CHANGELOG-3.0.md]: if I understand well, I would put that in VersionningBundle

File: CHANGELOG-3.0.md [hunk 3]
```diff
@@ -873,3 +873,23 @@
...
+- Move `Pim\Bundle\EnrichBundle\Normalizer\ViolationNormalizer` to `Akeneo\Pim\Enrichment\Component\Product\Normalizer\InternalApi\ViolationNormalizer`
+- Move `Pim\Bundle\EnrichBundle\Normalizer\AttributeOptionValueCollectionNormalizer` to `Akeneo\Pim\Structure\Component\Normalizer\InternalApi\AttributeOptionValueCollectionNormalizer`
+- Move `Pim\Bundle\EnrichBundle\Normalizer\AttributeOptionValueNormalizer` to `Akeneo\Pim\Structure\Component\Normalizer\InternalApi\AttributeOptionValueNormalizer`
+- Move `Pim\Bundle\EnrichBundle\Normalizer\VersionedAttributeNormalizer` to `Akeneo\Pim\Structure\Component\Normalizer\InternalApi\VersionedAttributeNormalizer`
+- Move `Pim\Bundle\EnrichBundle\Normalizer\FileNormalizer` to `Akeneo\Tool\Component\FileStorage\Normalizer\FileNormalizer`
```
Comment (jjanvier) [inline CHANGELOG-3.0.md]: if it's related to the file/image attribute that is used in a product value, I would put it in Akeneo\Pim\Enrichment\Component\Product

File: CHANGELOG-3.0.md (hunk 1 above)
Comment (arnolanglade) [reply CHANGELOG-3.0.md]: Fixed https://github.com/akeneo/pim-community-dev/pull/8864/files#diff-8ee894204f471eae7d606c828a1af52fR876

File: CHANGELOG-3.0.md (hunk 3 above)
Comment (arnolanglade) [reply CHANGELOG-3.0.md]: Fixed https://github.com/akeneo/pim-community-dev/pull/8864/files#diff-8ee894204f471eae7d606c828a1af52fR894

**Pull request merged**